frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGitMacheteRepositorySnapshot.java

```diff
@@ -88,8 +89,8 @@ public DemoGitMacheteRepositorySnapshot() {
     this.roots = List.of(root);
   }
 
-  static SyncToRemoteStatus getSTRSofRelation(SyncToRemoteStatus.Relation relation) {
-    return SyncToRemoteStatus.of(relation, "origin");
+  static RelationToRemote getSTRSofRelation(SyncToRemoteStatus syncToRemoteStatus) {
```

Comment:
Rename the method, you're now getting _relation of status_ after the renames, not _status of relation_ ;)